test/js/main.js

```diff
@@ -17,7 +17,8 @@ const bsCapabilitiesDefault = {
   'browserstack.key' : process.env.BROWSERSTACK_ACCESS_KEY,
   'browserstack.local' : 'true',
   "unexpectedAlertBehaviour": "dismiss",
-  "unexpectedPromptBehaviour": "dismiss"
+  "unexpectedPromptBehaviour": "dismiss",
+  'browserstack.ie.enablePopups' : 'false'
```

Comment:
sorry for the stupid question, but any reason we need to disable popups explicitly? (nothing against it, just curious of the reason)

Reply:
Some popups, not related to our SDK, have been disturbing our tests on Browserstack, mostly on IE11, so to allow for the smooth test execution we are blocking them.